src/components/MembersSectionNew/components/MembersCard/Presentation.tsx

```diff
@@ -0,0 +1,79 @@
+import { Box, Text, Flex, Button } from '@chakra-ui/react';
+import Image from 'next/image';
+
+import Socials from '../Socials';
+
+import { MemberType } from '../../types/MembersSection.type';
+
+import styles from './membersCard.module.css';
+import SettingButton from '../../../SettingButton/SettingButton';
+
+export default function MembersCardPresentation({
+  member,
+  openSkillUpdateModal,
+  openRoleUpdateModal,
+  shouldShowSetting,
+  hideSetting,
+  showSetting,
+}: {
+  member: MemberType;
+  openSkillUpdateModal: () => void;
+  openRoleUpdateModal: () => void;
+  shouldShowSetting: boolean;
+  hideSetting: () => void;
+  showSetting: () => void;
+}) {
+  return (
+    <Box
+      as='button'
+      onMouseEnter={showSetting}
+      onMouseLeave={hideSetting}
+      className={styles.member_card}
+    >
+      <Box className={styles.member_card__image_container}>
+        <Image
+          className={styles.member_card__image}
+          src={member?.picture?.url}
+          alt='Picture of the author'
+          fill
+        />
+      </Box>
+      <Text as='h1' className={styles.member_card__username}>
+        {`${member.first_name} ${member.last_name}`}
+      </Text>
+      <Flex justify='center' className={styles.member_card__socials}>
+        {member?.twitter_id && (
```

Comment:
this can be done with map where you check the keys on member object and then map over those